src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -65,6 +65,7 @@ internal interface ISyntaxFactsService : ILanguageService
         bool IsInvocationExpression(SyntaxNode node);
         bool IsExpressionOfInvocationExpression(SyntaxNode node);
         SyntaxNode GetExpressionOfInvocationExpression(SyntaxNode node);
+        SeparatedSyntaxList<TSyntaxNode> GetArgumentsOfInvocationExpression<TSyntaxNode>(SyntaxNode node) where TSyntaxNode : SyntaxNode;
```

Comment:
No.  The parameter is the InvocationExpression node.  The TypeParameter expresses the types of the Arguments you expect to get back.